src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiSampleDetails.tsx

```diff
@@ -0,0 +1,231 @@
+import { SwapOutlined, SwapRightOutlined } from "@ant-design/icons";
+import {
+  Avatar,
+  Cascader,
+  Checkbox,
+  Col,
+  Form,
+  FormInstance,
+  Input,
+  List,
+  Row,
+  Select,
+  Space,
+} from "antd";
+import React from "react";
+import { useLoaderData } from "react-router-dom";
+import {
+  DefaultModifiableField,
+  LoaderValues,
+  SampleRecord,
+} from "./CreateNcbiExport";
+
+const rules = [
+  {
+    required: true,
+    message: i18n("CreateNcbiExport.required"),
+  },
+];
+
+export default function CreateNcbiSampleDetails({
+  form,
+  sample,
+  onChange,
+}: {
+  form: FormInstance;
+  sample: SampleRecord;
+  onChange: () => void;
+}): JSX.Element {
+  const { strategies, sources, platforms, selections }: LoaderValues =
+    useLoaderData();
+
+  const onDefaultModifiableFieldChange = (field: DefaultModifiableField) => {
+    const value = form.getFieldValue(["samples", sample.name, field]);
+
+    // Update that the field was modified directly on the sample
+    form.setFieldsValue({
+      samples: {
+        [sample.name]: { [field]: { ...value, fromDefault: false } },
+      },
+    });
+    onChange();
+  };
+
+  const singles = sample.files.singles.map((file) => ({
+    label: (
+      <Space>
+        <Avatar
+          size="small"
+          style={{ backgroundColor: `var(--primary-grey)` }}
+          icon={<SwapRightOutlined />}
+        />
+        <span className="t-single-name">{file.name}</span> -{" "}
```

Comment:
Can we add the created dates here as well? Reasoning is if a user has multiple of the same named files then they can easily distinguish between an older and newer file. Same thing for the pairs below. For this we may want to change how the file details are displayed so that the file name, file size, and file created date aren't squished together

![fileswithsamename](https://user-images.githubusercontent.com/25466211/187999839-1ec9ae88-fd2d-49a1-9a49-9cd2f60e7d2b.png)

![fileswithsamenameExport](https://user-images.githubusercontent.com/25466211/187999888-181ecbcd-a948-4867-b52a-2fab9cf81480.png)

Reply:
Thanks for updating this. It looks really good.